test/test_mcmc.py

```diff
@@ -455,39 +478,9 @@ def model(y_obs):
 
     # Run MCMC on zero observations.
     kernel = NUTS(model)
-    mcmc = MCMC(kernel, 200, 200)
+    mcmc = MCMC(kernel, 300, 500, jit_model_args=True)
```

Comment:
Good idea. I have been trying to restrict pytest params since a lot of the test time is taken up by jax compilation. For instance, adding these params makes this test run in 1min from the earlier 10s. But I think its a good idea to aim for better coverage and then think about making tests faster. Btw..on jax 0.1.49, this was taking around 6-7 seconds, but my guess is that #463 has increased this time to 10s now.